src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.ServiceClientAdapter/ServiceClientAdapter.cs

```diff
@@ -24,26 +24,17 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.ServiceClient
 {
     public partial class ServiceClientAdapter
     {
-        const string AppSettingsSectionName = "appSettings";
         const string RecoveryServicesResourceNamespace = "Microsoft.RecoveryServices";
-        const string ProviderNamespaceKey = "ProviderNamespace";
         const string AzureFabricName = "Azure";
 
         ClientProxy<RecoveryServicesNS.RecoveryServicesBackupManagementClient, RecoveryServicesModelsNS.CustomRequestHeaders> BmsAdapter;
 
         public ServiceClientAdapter(SubscriptionCloudCredentials creds, Uri baseUri)
         {
-            System.Configuration.Configuration exeConfiguration = System.Configuration.ConfigurationManager.OpenExeConfiguration(System.Reflection.Assembly.GetExecutingAssembly().Location);
```

Comment:
Check with Venkatesh if we can push back this removal of appSettings. This is needed for automation of our testing with latest branch.